dvc/output.py

```diff
@@ -345,7 +345,7 @@ def __init__(
         # should be absolute and don't contain remote:// refs.
         self.stage = stage
         self.meta = meta
-        self._files = files
+        self.files = files
```

Comment:
This is part of the dvcfile, so it is okay to have it public, same as other fields.